lms/templates/admin/course/show.html.jinja2

```diff
@@ -27,8 +27,8 @@
 
     <fieldset class="box mt-6">
         <legend class="label has-text-centered">Assignments</legend>
-        {% if course.assignments %}
-            {{ macros.assignments_table(request, course.assignments) }}
+        {% if course._assignments %}
+            {{ macros.assignments_table(request, course._assignments) }}
```

Comment:
I would have expected this to not be possible, since the property is private 🤔 

I guess Python doesn't work as I though it did 😅